python/cli/record/oak.py

```diff
@@ -90,7 +90,7 @@ def record(args):
     outputFolder = args.output
     if args.auto_subfolders:
         import datetime
-        autoFolderName = datetime.datetime.now().strftime("%Y%m%dT%H%M%S")
+        autoFolderName = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
```

Comment:
Changed the Python recording tool's datetime -> string formatting a little (to match other wrappers). Or do you prefer the old version, and I should use that in the other wrappers?

Reply:
This is a good change.

Reply:
Agreed. The old format is the standard ISO8601 datetime format and can be parsed in many programming languages with some method that mentions that identifier (it can also optionally handle time zones etc. unambiguously). However, the proposed format is easier to read (by humans) and I think this is more relevant in this context.